### ISSUE TITLE
Hebrew / עברית

### ISSUE THREAD

**Issue by AlonDan** [jamiepine/voicebox#199]
Please add Hebrew support 🙏

**Comment by nadavox** [comment]
Hey I open PR for this please vote :
feat: add Hebrew language support with Chatterbox TTS and ivrit-ai Whisper
#194

**Comment by jamiepine** [comment]
This is now supported in **Voicebox v0.2.x**! 🎉

The new **Chatterbox Multilingual** engine brings zero-shot voice cloning to **23 languages**: Arabic, Chinese, Danish, Dutch, English, Finnish, French, German, Greek, Hebrew, Hindi, Italian, Japanese, Korean, Malay, Norwegian, Polish, Portuguese, Russian, Spanish, Swahili, Swedish, and Turkish.

Just select **Chatterbox** from the engine dropdown and pick your language — the language list dynamically filters to show only what the selected engine supports.

**To get the update:**
- Download the latest release from [voicebox.sh](https://voicebox.sh) or update from within the app
- If you're already on v0.1.x, the app should prompt you to update automatically

Thanks for the request — closing this as resolved. If the language support doesn't meet your needs, feel free to open a new issue with details!

**Comment by AlonDan** [comment]
Thanks for the hard work!
I appreciate this project, already gave a STAR ⭐with much love.

Sadly, it gets error when trying to download in the latest version, I'll wait for the update then I'll see if it works with decent Hebrew.
Please keep up the good work! 💙

**Comment by AlonDan** [comment]
Dear @jamiepine here is an Hebrew testing update: (I hope this feedback helps somehow)

Thanks for the last update, I checked out the latest Chatterbox in Hebrew and it is VERY BAD,
the quality is nice, but I have no idea what they trained on, it's like 70%-80% wrong way of how anything pronouncing or sound.
for comparison, VibeBox even out of the box (without LoRA training) is about 90% very accurate when using Hebrew.

The problem with VibeVoice compare to Qwen and Chatterbox is that the Emotion Tags are not working as accurate or controllable so the quality and accuracy is great, but almost no control if you want to be very precise with emotions via tags.

--

Also, a Suggestion:
Will you consider to allow Finetuning / LoRA training within VoiceBox? 
If so, I will be able to try training Hebrew for Qwen / Chatterbox to make it much better.

From my minor experience with Open Voice Studio repo, it took few minutes to train on my RTX 5090 32GB VRAM.
About 15-20 minutes to train a VibeVoice 7B LoRA with around 60 audio clips, each 3-15 seconds, total length of about 6-7 minutes of audio, the results were AMAZING! 
But installing Open Voice Studio for RTX 5090 wasn't simple (because of the specific sm120, python 3.10, pytorch 12.8 etc..) consider I'm not very technical it took me some time to make it work on my RTX 5090 locally thanks to Gemini AI.

If you will be able to add a TRAINING section to VoiceBox in the same friendly updates / installations you did so far in 0.2.3 (latest version I've tried) it will be AMAZING and I'll be happy to share my feedback to help you improve it.
